app/funcs/csv_reader.py

```
@@ -80,11 +80,65 @@ def read_csv(self, file: Path, **kwargs) -> Generator[Dict[int, Dict], None, Non
+            print(f"Error reading parquet file {file}: {str(e)}")
+            # Try alternative approach if scan_parquet fails
+            try:
+                df = pl.read_parquet(file, **kwargs)
+                for row_dict in df.to_dicts():
+                    yield row_dict
+            except Exception as e2:
+                print(f"Alternative approach also failed for {file}: {str(e2)}")
+
+    @include_file_origin
+    @include_download_date
+    def read_txt_file(self, file: Path, **kwargs) -> Generator[Dict[int, Dict], None, None]:
```

Ruff (0.14.7)

Missing type annotation for `**kwargs`

(ANN003)